tools/clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -365,8 +367,9 @@ llvm::Type *CodeGenTypes::ConvertType(QualType T) {
                  .getConstantArrayType(eltTy, llvm::APInt(32, count),
                                        ArrayType::ArraySizeModifier::Normal, 0)
                  .getTypePtr();
-      }
-      else
+      } else if (hlsl::IsHLSLHitObjectType(T)) {
+        return hlsl::dxilutil::GetHLSLHitObjectType(&TheModule);
+      } else
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
      } else if (hlsl::IsHLSLHitObjectType(T))
        return hlsl::dxilutil::GetHLSLHitObjectType(&TheModule);
      else
```

Reply:
Applied in [ba11d4f](https://github.com/microsoft/DirectXShaderCompiler/pull/7097/commits/ba11d4f3ffde25c5945852099caf9e983b2deb58)